organizers/2022-08-16.html

```diff
@@ -0,0 +1,165 @@
+---
+layout: organizers
+---
+
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>August 16th, 2022</h2>
+    <img data-src="/img/django-logo-white-transparent.svg"
+         style="width: 40%; background: none;"
+         alt="Django logo"
+         class="plain">
+    <h2>Talks starting at 19:00!</h2>
+</section>
+
+<section>
+    <h1>👋 Welcome All!</h1>
+</section>
+
+<section>
+    <h1>Code of conduct</h1>
+    <h2>😃 Be Awesome to Each Other</h2>
+    <p>
+        The mission of The London Django Meetup Group is to provide a fun,
+        welcoming and professional environment for a diverse group of people.
+    </p>
+    <p>
+        Code of conduct:
+        <a href="https://www.djangolondon.com/code-of-conduct/">djangolondon.com/code-of-conduct</a>
+    </p>
+</section>
+
+<section data-background="#eee">
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Kraken Tech (part of the Octopus Energy Group)</strong></li>
+        <li><strong>Jetbrains</strong></li>
+    </ul>
+</section>
+
+<section data-background="#eee">
+    <img data-src="/img/2022_KrakenV2_stacked_White.png"
+         style="width: 60%;"
+         alt="Kraken Tech"
+         class="plain">
+    <p>
+        <em>
+            Kraken is a revolutionary energy tech platform built with Django. Serving 25 million households worldwide.
+        </em>
+    </p>
+</section>
+
+<section>
+    <h1>📋 Today's Schedule</h1>
+    <h2>🐍 News</h2>
+    <h2>🎤 Talk</h2>
+    <h2>🎁 Prize Draw</h2>
+    <h2>💬 Open Discussion</h2>
+</section>
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>Security Release</h2>
+    <p>3rd Aug</p>
+    <p>Versions 4.0.7 and 3.2.15.</p>
+    <p>Potential reflected file download vulnerability in FileResponse.</p>
+</section>
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>Django 4.1 released</h2>
+    <p>3rd Aug</p>
+    <p>An async interface to the ORM, and the ability to define async handlers on class-based views.</p>
+    <p>The use of ORM-defined database constraints in model validation.</p>
+    <p>Better form rendering accessibility and output style customization.</p>
+</section>
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>Django Paris Meetup Group</h2>
+    <p>Looking for speakers.</p>
+    <p>Event TBC but something will happen - see Twitter.</p>
+</section>
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>PyCon UK</h2>
+    <p>Cardiff, 16th-18th September</p>
+</section>
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>DjangoCon Europe 2022</h2>
+    <p>Porto, 21st-23rd September</p>
+    <p>DjangoGirls workshop on 24th September</p>
+</section>
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>DjangoCon US 2022</h2>
+    <p>San Diego, 16-21st October!</p>
```

Comment:
ordinals for symmetry
```suggestion
    <p>San Diego, 16th-21st October!</p>
```